zones/wip.la.yaml

```diff
@@ -321,4 +321,3 @@ zzhsjksk: [NS1.AFRAID.ORG, NS2.AFRAID.ORG, NS3.AFRAID.ORG, NS4.AFRAID.ORG]
 uy-1: [ns1.desec.io, ns2.desec.org, ns1.desec.io, ns2.desec.org]
 connect: [ns1.desec.io, ns2.desec.org, ns1.desec.io, ns2.desec.org]
 higgs: [ns1.cloudns.net, ns2.cloudns.net, ns3.cloudns.net, ns4.cloudns.net]
-nt: [ns1.digitalocean.com, ns2.digitalocean.com, ns3.digitalocean.com]
```

Comment:
You should not remove other domains, @ezcolombo.

Reply:
Use GitHub.dev to edit without trailing Newline. See #519 and #1044 for more information.